src/js/blocks/form-fields/settings.js

```diff
@@ -158,10 +158,6 @@ const settingsField = {
 			type: 'boolean',
 			__default: false,
 		},
-		uuid: {
```

Comment:
cool cool,so custom-fields needs to be updated

Reply:
I was getting there (you beat me to it)